doc/demo-beta.1/README.md

```diff
@@ -0,0 +1,218 @@
+LNP-node demo
+===
+
+### Introduction
+
+This document contains demonstration of `lnp-node` functionality as for [version 0.1.0-beta.1](https://github.com/LNP-BP/lnp-node/releases/tag/v0.1.0-beta.1) and mainly follows [this video-recorded demo]( https://www.youtube.com/watch?v=ns58fUlyd5o).
+
+Two different setups are available:
+- [local installation](#local)
+- [docker](#docker)
+
+Once either of them is complete, you can proceed with the actual [demo](#demo)
+
+## Local
+
+This setup consists in a local installation of `lnp-node`.
+
+#### Requirements
+
+- [cargo](https://doc.rust-lang.org/book/ch01-01-installation.html#installation)
+- [git](https://git-scm.com/downloads)
+
+Furthermore, you will need to install a number of system dependencies:
+```bash=
+sudo apt install -y build-essential pkg-config libzmq3-dev libssl-dev libpq-dev libsqlite3-dev cmake
+```
+### Setup
+
+Alongside `lnp-node`, we also need to install `rgb-node` which will be queried for token-related data, then we can launch it in the first terminal (ctrl+C to stop it):
+```bash=
+git clone https://github.com/LNP-BP/lnp-node
+cd lnp-node/doc/demo-beta.1
```

Comment:
Git is not required, the installation is done from the remote source (crates.io)
```suggestion
mkdir demo
cd demo
```

Reply:
Also in the demo I use the `samples/source_tx.psbt` that is in git repo. I could also advice to create a psbt in core and copypaste it somewhere, or include an hardcoded one in the demo, but this looked like the most newbie-proof approach

Reply:
then you are right, we need to clone the repo. However in this case we also need to clone RGB repo, since samples are there

Reply:
I added samples to the PR to keep paths consistent with the docker version and have the bigger part of the demo equal for the two setups, since the samples directory is taken as a docker volume in the compose. Sure we can also change it to clone RGB-node repo and get samples from there, but it would probably be a bit of a hassle to figure out relative paths for docker-compose to find the repo, and a small deviation from the guide would cause a user to end up in errors and headaches.

Reply:
Not sure I follow. For now we are cloning LNP repo w/o any reason for that, so we can change on installing from crates.io. So basically we need either to accept my suggestion - or clone RGB git repo as well

Reply:
There is just one reason why I included the lnp-node repo clone in the local installation: getting the `lnp-node/doc/demo-beta.1/samples` directory. Installation via `cargo --git` does not require cloning the repo anyways. I discussed "installing from crates" in https://github.com/LNP-BP/lnp-node/pull/35#discussion_r540359603

Following the docker setup, user needs to clone lnp-node repo to get docker-related files.
The reason why I decided to have the `samples` directory in lnp-node repo, and not suggesting to clone rgb-node repo instead, is to have an easier docker setting. In fact `samples` can be loaded as a docker volume and used to 1) provide rgb-node with a sample psbt 2) share consignment to lnp-node container. 
Since `samples` directory is available in lnp-node repo for docker requirements, I see no point in doing something different for the local setup, as it would cause relative paths to the samples directory to be different across the two setups and complicate a bit the demo, while bringing no advantages.

Reply:
Sorry, I forgot that there are `samples` in LNP Node as well. Then it's ok to leave it as it is